stdlib/public/TensorFlow/TensorHandle.swift

```diff
@@ -121,14 +131,21 @@ extension TensorHandle : TensorSendableReceivable {
       dumpTensorContent(self.cTensorHandle, Scalar.self)
     }
     let status = TF_NewStatus()
-    let cTensor = TFE_TensorHandleResolve(self.cTensorHandle, status)
-    checkOk(status)
-    TF_EnqueueNamedTensor(
-      computation.cSession, Int32(tensorId), cTensor, status)
+    internalConsistencyCheck(status != nil)
+    if _RuntimeConfig.usesTFEagerAPI {
+      let context = _ExecutionContext.global
+      TFE_EnqueueNamedTensorFromCtx(
+        context.eagerCtx, Int32(tensorId), self.cTensorHandle, status)
+    } else {
+      let cTensor = TFE_TensorHandleResolve(self.cTensorHandle, status)
```

Comment:
`self.` is unnecessary. Though it doesn't matter, we've been consistently avoiding unnecessary `self.` so it'd be nice to maintain that consistency.

Reply:
Done.